http_parser.c

```diff
@@ -123,7 +123,7 @@ do {                                                                 \
     FOR##_mark = NULL;                                               \
   }                                                                  \
 } while (0)
-  
```

Comment:
Grr.. Thanks. Missed those. Looks like I need to tweak my atom editor
settings so it's not doing these silly style edits
On Sep 27, 2015 12:57 PM, "Fedor Indutny" <notifications@github.com> wrote:

> In http_parser.c
> <https://github.com/joyent/http-parser/pull/266#discussion_r40508903>:
>
> > @@ -123,7 +123,7 @@ do {                                                                 \
> >      FOR##_mark = NULL;                                               \
> >    }                                                                  \
> >  } while (0)
> > -
>
> Unnecessary style change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joyent/http-parser/pull/266/files#r40508903>.
>